connectors/src/connectors/slack/bot.ts

```diff
@@ -232,18 +334,16 @@ async function botAnswerMessage(
     slackAvatar: slackUserInfo.image_512 || null,
     channelId: slackChannel,
     messageTs: slackMessageTs,
-    threadTs:
-      slackThreadTs || lastSlackChatBotMessage?.threadTs || slackMessageTs,
+    threadTs: slackThreadTs || slackMessageTs,
     conversationId: lastSlackChatBotMessage?.conversationId,
     userType: slackUserInfo.is_bot ? "bot" : "user",
   });
 
   const buildContentFragmentRes = await makeContentFragment(
     slackClient,
     slackChannel,
-    lastSlackChatBotMessage?.threadTs || slackThreadTs || slackMessageTs,
+    slackThreadTs || slackMessageTs,
```

Comment:
Can you explain why we remove `lastSlackChatBotMessage?.threadTs`?

Reply:
it's defined higher as : 
```    
lastSlackChatBotMessage = await SlackChatBotMessage.findOne({
      where: {
        connectorId: connector.id,
        channelId: slackChannel,
        threadTs: slackThreadTs,
      },
      order: [["createdAt", "DESC"]],
      limit: 1,
    });
```
with: `where : {threadTs: slackThreadTs}`, meaning astSlackChatBotMessage?.threadTs is always undefined or equals to slackThreadTs.